packages/semver/src/executors/version/version.ts

```diff
@@ -34,27 +36,28 @@ export function versionWorkspace({
   skipProjectChangelog: boolean;
   workspaceRoot: string;
 } & CommonVersionOptions) {
-  return concat(
-    getProjectRoots(workspaceRoot).pipe(
-      switchMap((projectRoots) =>
-        _generateProjectChangelogs({
-          workspaceRoot,
-          projectRoots,
-          ...options,
-        })
-      ),
-      /* Run Git add only once, after changelogs get generated in parallel. */
-      switchMap((changelogPaths) =>
-        addToStage({ paths: changelogPaths, dryRun: options.dryRun })
-      )
+  return getProjectRoots(workspaceRoot).pipe(
+    concatMap((projectRoots) =>
+      _generateProjectChangelogs({
+        workspaceRoot,
+        projectRoots,
+        ...options,
+      })
     ),
-    getPackageFiles(workspaceRoot).pipe(
-      switchMap((packageFiles) =>
-        _runStandardVersion({
-          bumpFiles: packageFiles,
-          skipChangelog: skipRootChangelog,
-          ...options,
-        })
+    /* Run Git add only once, after changelogs get generated in parallel. */
+    concatMap((changelogPaths) =>
+      addToStage({ paths: changelogPaths, dryRun: options.dryRun })
+    ),
+    reduce(noop),
```

Comment:
This reduces the stream from one emission per project down to just a single emission.

Reply:
Perfect! Nice refactor